modules/storage-api/src/main/java/org/apache/ignite/internal/storage/index/IndexStorage.java

```diff
@@ -67,7 +67,7 @@ public interface IndexStorage {
     @Nullable RowId getNextRowIdToBuild() throws StorageException;
 
     /**
-     * Sets the row ID for which the index needs to be built, {@code null} means that the index is built.
+     * Sets the row ID for which the index needs to be build, {@code null} means that the index is build.
```

Comment:
Let's revert this change, because 'to be built' seems to be the correct form ('built' is the 3rd form for 'to build')